src/types.ts

```diff
@@ -45,38 +57,45 @@ export enum IssueTimeField {
   ClosedAt,
 }
 
-export interface ReleasesOptions {}
-export interface TagsOptions {}
-export interface PullRequestsOptions {
+export interface Indexable {
```

Comment:
um... this kinda sidesteps all value in using typescript!

Reply:
yeah, that's the only way I could accomplish this (as far as I know :\ )
https://github.com/grafana/github-datasource/pull/42/files#diff-f65c1b6588a53320c1abf59e0caed96cR9-R26

any suggestions?